runtime/moonbase/tests/common/mod.rs

```diff
@@ -74,9 +74,22 @@ pub fn evm_test_context() -> evm::Context {
 	}
 }
 
+pub fn asset_id_to_address(asset_id: AssetId) -> H160 {
```

Comment:
Is this duplicated with the conversion code in the runtime?

Reply:
yes good catch, already fixed it